src/leaderboard/leaderboard.js

```diff
@@ -0,0 +1,82 @@
+import './leaderboard.css';
+
+import Button from '../components/Button/Button';
+import '../components/Button/Button.css';
+
+// funkcja renderująca stronę
+
+export function renderLeaderboard() {
+  podium();
```

Comment:
podium function name is not descriptive (we do not know what it does) - use eg. renderPodium